core/src/components/searchbar/searchbar.ionic.scss

```diff
@@ -28,36 +28,15 @@
 // -----------------------------------------
 
 .searchbar-search-icon {
-  // Position is based on the size of the search icon.
-  @include globals.position(globals.$ion-scale-400, null, null, globals.$ion-scale-400);
-
-  width: globals.$ion-scale-400;
-  height: globals.$ion-scale-400;
+  width: globals.$ion-scale-500;
+  height: globals.$ion-scale-500;
```

Comment:
The Figma design for this shows it should be scale-400 but actually uses scale-500. Have we asked design to clarify this?